expyfun/analyze/_viz.py

```diff
@@ -368,7 +368,7 @@ def barplot(h, axis=-1, ylim=None, err_bars=None, lines=False,
             # raise apex a bit extra if prev bracket label centered on bar
             prev_label_pos = brk_c[_ix - 1] if _ix else -1
             label_bar_ix = np.where(np.isclose(bar_centers, prev_label_pos))[0]
-            if label_bar_ix in _brk:
+            if any(np.array_equal(label_bar_ix, x) for x in _brk):
```

Comment:
@drammock this fixes a bug I got in testing where I got this warning:
```
/home/larsoner/Documents/python/expyfun/expyfun/analyze/_viz.py:371: DeprecationWarning: elementwise comparison failed; this will raise the error in the future.
  if label_bar_ix in _brk:
```
because it was in this state:
```
(Pdb) p _brk
([0, 1, 2], 3)
(Pdb) p label_bar_ix
array([], dtype=int64)
```
I hope this is okay.